app/models/post.rb

```diff
@@ -1,14 +1,20 @@
 class Post < ApplicationRecord
-  has_many :comments
-  has_many :likes
   belongs_to :author, class_name: 'User'
-  before_save :post_count
+  has_many :likes
+  has_many :comments
+  after_save :count_posts
+
+  validates :title, presence: true, length: { in: 1..250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
 
-  def recent_comments
+  def most_recent_comments
     comments.order(created_at: :desc).limit(5)
   end
```

Comment:
- [ ] Kindly consider testing this method as requested by the previous reviewer. We want to know if the 5 most recent comments are being returned on each post.

Reply:
I also added the test.